cql3/expr/expression.cc

```diff
@@ -864,6 +946,30 @@ std::ostream& operator<<(std::ostream& os, const expression& expr) {
             [&] (const column_value_tuple& tuple) {
                 fmt::print(os, "({})", fmt::join(tuple.elements, ","));
             },
+            [&] (const unresolved_identifier& ui) {
+                fmt::print(os, "unresolved({})", *ui.ident);
+            },
+            [&] (const column_mutation_attribute& cma)  {
+                fmt::print(os, "{}({})",
+                        cma.kind == column_mutation_attribute::attribute_kind::ttl ? "TTL" : "WRITETIME",
+                        *cma.column);
+            },
+            [&] (const function_call& fc)  {
+                std::visit(overloaded_functor{
+                    [&] (const functions::function_name& named) {
+                        fmt::print(os, "{}({}})", named, fmt::join(fc.args, ", "));
+                    },
+                    [&] (const shared_ptr<functions::function>& anon) {
+                        fmt::print(os, "<anonymous function>({}})", fmt::join(fc.args, ", "));
+                    },
+                }, fc.func);
+            },
+            [&] (const cast& c)  {
+                fmt::print(os, "({}} AS {})", *c.arg, c.type);
+            },
+            [&] (const field_selection& fs)  {
+                fmt::print(os, "({}}.{})", *fs.structure, fs.field);
```

Comment:
There are more `}`s than `{`s in the format string -- is that intentional?  What does it mean?

Reply:
Just a bug.

Reply:
Note it also appears in other cases above.

Reply:
At least I'm consistent